docs/environment_variables.md

```diff
@@ -30,23 +30,36 @@ with httpx.Client() as client:
     r = client.get("https://google.com")
 ```
 
+Debug output:
+
+```console
+$ HTTPX_LOG_LEVEL=debug python test_script.py
+DEBUG [2019-11-02 23:39:09] httpx.client - HTTP Request: www.google.com - "GET / HTTP/2" 200 OK
```

Comment:
I'm wondering how/if we should indicate http/https here?

Options:

* `HTTP Request: https://www.google.com - "GET / HTTP/2" 200 OK` # Include it in the origin.
* `HTTPS Request: www.google.com - "GET / HTTP/2" 200 OK` - Include it in the message itself.
* `HTTP Request: "GET https://www.google.com/ HTTP/2" 200 OK` - Include it in the request line.

Reply:
That might be useful, yeah. In general I don't think logs do include HTTPS, though (e.g. the current format is inspired by the [Common Log Format](https://httpd.apache.org/docs/1.3/logs.html#common), and that doesn't show the HTTP scheme). So I'm more inclined towards including it in the message.

Reply:
"the current format is inspired by the Common Log Format, and that doesn't show the HTTP scheme". Yeah I'm not sure here. Logging outgoing HTTP requests isn't neccessarily the same as logging requests to a webserver. (Eg. we want to see the host up front and center, wheras in the server case you typically don't need to see that.)

Reply:
Yeah sure, it's not like these debug logs are designed to be ingested by production log systems anyway. :) I prefer first option then — include it in the origin.

Reply:
Or actually, option 3 looks sensible too. Especially if we want to make it *not* look like a proper log line.

Reply:
I'm keen on 3, personally. 👍

Reply:
Same for me, 3 looks good.

Reply:
Cool, changed to use 3 then!